andela/valhalla-rc
# #158755792  improve image gallery

#### What does this PR do?
Improve image gallery

#### Description of Task to be completed?
Change the background color of product detail page
Change the `Add to cart` button color
Implement image hover for a clearer view of the product image

#### How should this be manually tested?
clone the project
cd into the project
run `meteor npm install` in your terminal
run `reaction`
check  http://localhost:3000/

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
Story Id: 158755792
Story Type: Feature
Story: Improve image gallery

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-07-18 at 12 00 09 pm" src="https://user-images.githubusercontent.com/14803275/42886417-6cedf540-8a9b-11e8-8a2b-601dbdb172f4.png"><img width="1395" alt="screen shot 2018-07-18 at 3 00 13 pm" src="https://user-images.githubusercontent.com/14803275/42886460-86b12f1a-8a9b-11e8-9cb3-d828380cd5c2.png">

#### Questions:
